cluster/gke/config-default.sh

```diff
@@ -42,3 +42,7 @@ ENABLE_L7_LOADBALANCING="${KUBE_ENABLE_L7_LOADBALANCING:-glbc}"
 ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-standalone}"
 
 KUBE_DELETE_NETWORK=${KUBE_DELETE_NETWORK:-false}
+
+# Indicates if the values (eg. kube password) in metadata should be treated as
+# canonical, and therefore disk copies ought to be recreated/clobbered.
+METADATA_CLOBBERS_CONFIG=true
```

Comment:
We can set this specifically to true in GKE because we know that the customer has not added an additional accounts to this file as they have no access to this file So it is safe to make this true in GKE. The customer should not be directly modifying this file in GCE, but we cannot rely on the fact that hey cannot to turn this feature on in GCE. This feature is intentionally turned on in GKE only.